src/Distortion.cpp

```diff
@@ -324,5 +395,16 @@ void applyDistortion(double ux, double uy, double &dx, double &dy,
         dy = ydistorted;
       }
     }
+    break;
+    // The dawn distortion model is "reversed" so the apply function computes
+    // the distorted focal plane and stores it into the distorted variables (dx, dy)
```

Comment:
Like above, I understand the situation, but found this comment confusing. The DAWNFC case under applyDistortion still calculates the distorted coefficients using the undistorted coefficients, it just doesn't need to use an iterative solution to do so.